### PR TITLE
Drawer navigation

### PR DESCRIPTION
**Pull request by Yennie-Leek**
My reflections on how this week's project turned out:
This was a bit of a struggle. Started with technical issues and couldn't start working with the code until later in the week. When I finally got it up and running I was a bit drained and confused what to actually do and how to start. But it worked out with some help from one friend in my team. 
I still have technical issues and cant connect to expo in my mobile, which also means I cant deploy it there. 
Hopefully I can have a look at it later and make it work. 

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: screens/ChuckNorris.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  useEffect (() => {
+    fetchChuckNorrisList()
+  }, [])
+
+  const fetchChuckNorrisList = () => {
+    fetch('https://api.chucknorris.io/jokes/random')
+    .then(res => res.json())
+    .then(value => setChuckNorris(value))
+    .catch(err => console.error(err))
```
Comment (KarolCislowski) [inline screens/ChuckNorris.js:46]: Remember to remove all console outputs before requesting the pull.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
I couldn't run your app. 😭  These are error messages that I saw on my device: 
image

image

**Comment by Yennie-Leek** [comment]
Hey! I have now created a snack so you will be able to view it live. I hope this will work as I still have issues to make the Expo work.. 
Link to my snack:
[https://snack.expo.io/@yennie/week-13-react-native](url)

**Comment by Yennie-Leek** [comment]
Hey!
I have now created a snack so you will be able to view it live. I hope this will work as I still have issues to make the Expo work..
Link to my snack:
***@***.***/week-13-react-native

Kind regards, Yennie

On 18 Apr 2021, at 14:39, Karol Cislowski ***@***.******@***.***>> wrote:



@KarolCislowski requested changes on this pull request.

I couldn't run your app. 😭 These are error messages that I saw on my device:
[image]<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F50926159%2F115145786-a53e8600-a053-11eb-946e-13351ec2fd98.png&data=04%7C01%7C%7Cdb9f0ff1b6c94fe316c908d9026704c8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637543463742836964%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=VZyuGh57vLksIlTEYU%2FWc8w0I4UrR4bnSvi5AqPH0jg%3D&reserved=0>

[image]<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F50926159%2F115145781-9a83f100-a053-11eb-879a-8a9670eaedaf.png&data=04%7C01%7C%7Cdb9f0ff1b6c94fe316c908d9026704c8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637543463742836964%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=rzrJk%2FHS1KTsMgJ8Tr3RWRSJOlqFHsyuFxc9gUeT1EU%3D&reserved=0>

________________________________

In screens/ChuckNorris.js<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-react-native-app%2Fpull%2F156%23discussion_r615390808&data=04%7C01%7C%7Cdb9f0ff1b6c94fe316c908d9026704c8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637543463742846961%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=UfGEZxIGFz3TbIA%2BCz2viJ9GY4FnTMZnOoNizl8t8Ko%3D&reserved=0>:

> +  font-size: 20px;
+`
+
+export const ChuckNorris = ({navigation}) => {
+
+  const [chuckNorris, setChuckNorris] = useState ('')
+
+  useEffect (() => {
+    fetchChuckNorrisList()
+  }, [])
+
+  const fetchChuckNorrisList = () => {
+    fetch('https://api.chucknorris.io/jokes/random')
+    .then(res => res.json())
+    .then(value => setChuckNorris(value))
+    .catch(err => console.error(err))


Remember to remove all console outputs before requesting the pull.

**Pull request closed without merging**